encryptor/config/schemaStore_test.go

```diff
@@ -1130,6 +1130,16 @@ schemas:
         tokenized: false
         token_type: str
     `},
+		{"tokenized: true without token_type",
```

Comment:
lets check with empty value too?)
```
tokenized: true
token_type: ""
```
To cover case with only `!= nil` checks (that we validate not only with such conditions)